eole/config/run.py

```diff
@@ -93,6 +93,10 @@ class PredictConfig(
         None  # patch for CT2 inference engine (to improve later)
     )
     model: ModelConfig | None = None
+    optional_eos: List[str] | None = Field(
```

Comment:
why don't we put this in the same section as bos_token/eos_token ?
It's better within the model I think.

Reply:
second thought should be fine here